ci/aws/aws-cluster.lokocfg.envsubst

```diff
@@ -99,13 +119,33 @@ storage:
       group:
         id: 500
 EOF
+      ,
+      <<EOF
+storage:
+  files:
+    - path: /etc/modules-load.d/nf.conf
+      filesystem: root
+      mode: 0644
+      contents:
+        inline: |
+          nf_conntrack
+    - path: /etc/sysctl.d/nf.conf
+      filesystem: root
+      mode: 0644
+      contents:
+        inline: |
+          net.netfilter.nf_conntrack_max=50000
+EOF
+      ,
     ]
   }
 
   # Adds oidc flags to API server with default values.
   # Acts as a smoke test to check if API server is functional after addition
   # of extra flags.
   oidc {}
+
+  conntrack_max_per_core = 0
```

Comment:
Q: Why is it set to zero here and in CLC it is set to something else?

Reply:
See flag documentation :) 0 disables kube-proxy tinkering the sysctl value, so then you can set it with CLC snippet. Otherwise kube-proxy would revert it to the default value.

Reply:
A comment on this would be helpful as to why it exists here.